errata_tool/product_version.py.orig

```diff
@@ -0,0 +1,93 @@
+from errata_tool import ErrataConnector
```

Comment:
I don't think you meant to git add this and the `.rej` file :)

Reply:
Yeah, removed it.